src/order/filter/at_top_bot.lean

```diff
@@ -448,5 +448,42 @@ map_at_top_eq_of_gc (λb, b * k + (k - 1)) 1
     calc b = (b * k) / k : by rw [nat.mul_div_cancel b hk]
       ... ≤ (b * k + (k - 1)) / k : nat.div_le_div_right $ nat.le_add_right _ _)
 
+lemma tendsto_at_top_of_monotone {ι  : Type*} [nonempty ι] [semilattice_sup ι]
+  {α : Type*} [linear_order α] {u : ι → α}
+  (h : monotone u) (H : ¬bdd_above (range u)) : tendsto u at_top at_top :=
```

Comment:
Note that we have `tendsto_at_top_at_top_of_monotone`.

Reply:
Do you want me to use it to simplify the proof or do you think that makes this lemma useless ?

Reply:
I guess Yury want to simplify here.

Reply:
I guess Yury wants to simplify.